src/index.html

```diff
@@ -1,13 +1,328 @@
+              </article>
+
+              <article class="sound-vision__product" id="headphones">
+                <a href="#earphones" class="sound-vision__link">
+                  <img
+                    src="./images/products/earphones-01.png"
+                    alt="Earphones"
+                    class="sound-vision__image"
+                  >
+                </a>
+                <a href="#earphones" class="sound-vision__link">
+                  <img
+                    src="./images/products/earphones-02.png"
+                    alt="Earphones"
+                    class="sound-vision__image"
+                  >
+                </a>
+                <h3 class="sound-vision__title">Earphones</h3>
+              </article>
+
+              <article class="sound-vision__product" id="televisions">
+                <a href="#smarts-home" class="sound-vision__link">
+                  <img
+                    src="./images/products/smarts-home-01.png"
+                    alt="Smart Home"
+                    class="sound-vision__image"
+                  >
+                </a>
+
+                <a href="#smarts-home" class="sound-vision__link">
+                  <img
+                    src="./images/products/smarts-home-02.png"
+                    alt="Smart Home"
+                    class="sound-vision__image"
+                  >
+                </a>
+
+                <h3 class="sound-vision__title">Smart Home</h3>
+
+                <a href="#viwe-all" class="sound-vision__button button">View all</a>
+              </article>
+            </div>
+          </section>
+        </div>
+      </div>
+
+      <section class="world-bo" id="world-of-BO">
+        <div class="world-bo__background"></div>
+
+        <div class="container">
+          <div class="main__content">
+            <div class="world-bo__content">
+
+              <h2 class="world-bo__title section-title">Timeless, for 50 years and counting</h2>
+
+              <p class="world-bo__description">
+                Bang & Olufsen is expanding its Recreated Classics
+                Program with the launch of Beosystems – a limited
+                edition music system that transcends time by
+                bridging the gap between one of our iconic
+                designs from 1972 and today&apos;s cutting-edge
+                digital technology.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__content">
+          <section class="contact-us">
+            <h2 class="section-title">Contact us</h2>
+
+            <div class="contact-us__wrapper">
+              <form
+                onsubmit="this.reset(); return false;"
+                action="#"
+                method="get"
+                class="contact-us__form"
+              >
```

Comment:
all fields should be required, 13'th point in checklist

Reply:
![image](https://github.com/mate-academy/layout_landing-page/assets/108736212/e5cc8f34-d4db-446e-8203-32471d9cf027)

![image](https://github.com/mate-academy/layout_landing-page/assets/108736212/97a60381-45ea-4776-be33-3ba08de570e0)

on the second screenshot, you can see the cursor, this shows how the hover looks like